packages/plugin-loansresearch-ui/src/types.ts

```
@@ -0,0 +1,29 @@
+import { IAttachment } from '@erxes/ui/src/types';
+
+export type IIncome = {
+  _id?: string;
+  incomeType: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)